uvm/api/com/untangle/uvm/UriTranslation.java

```diff
@@ -48,4 +48,31 @@ public String toJSONString()
         return jO.toString();
     }
 
+    /**
+     * Perform value comparision between this object and another
+     * UriTranslation object.
+     *
+     * @param ot Other UriTranslation object to compare.
+     * @return true if all values match, otherwise false.
+     */
+    public boolean equals(UriTranslation ot)
+    {
+        if( ( getUri() == null && ot.getUri() != null ) ||
```

Comment:
Maybe check ot != null first to prevent exception if null gets passed to the function